README.md

```diff
@@ -130,13 +131,14 @@ Specific to Copernicus Sentinel data discovery, download and processing.
 - [**`Sen2-Agri`**](https://github.com/Sen2Agri/Sen2Agri-System)
   - toolbox for processing images for agricultural purposes
   - includes modules for atmospheric correction, monthly syntheses, biophysical variables, crop mask, crop-type classification and an [orchestrator](http://www.esa-sen2agri.org/operational-system/system-description/)
-- [**`sentinel2-cloud-detector`**](https://github.com/sentinel-hub/sentinel2-cloud-detector)
+- [**`s2cloudless`**](https://github.com/sentinel-hub/sentinel2-cloud-detector)
   - single scene, pixel-based cloud detection algorithm used at [Sentinel-Hub](https://www.sentinel-hub.com/)
   - [accompanying write-up](https://medium.com/sentinel-hub/improving-cloud-detection-with-machine-learning-c09dc5d7cf13) with performance comparison to other cloud detection algorithms
 - [**`Sen2Cor`**](http://step.esa.int/main/third-party-plugins-2/sen2cor/)
   - atmospheric correction of Sentinel-2 data
   - basis for [L2A](https://sentinel.esa.int/web/sentinel/technical-guides/sentinel-2-msi/level-2a/algorithm) data published on Copernicus Open Access Hub
-
+- [**`sentinelgif`**](https://github.com/fredliporace/sentinelgif)
```

Comment:
Does this still work? It is Python 2, two years old, linked to the old sat api and wants to pull data from the not-so-public-anymore AWS S2 repo just with `/vsicurl`...

Reply:
You're right. I missed in my notes that I tested it more than a year ago. Thanks for the hint!
```suggestion
```